tests/test_magelib/test_document.py

```diff
@@ -111,43 +111,43 @@ def test_cross_referencing(self):
         doc.annotate_entity(field_name="chunks", entities=chunks)
 
         # find by span is the default overload of Entity.__attr__
-        self.assertListEqual(doc.chunks[0].tokens, tokens[0:3])
-        self.assertListEqual(doc.chunks[1].tokens, tokens[3:5])
-        self.assertListEqual(doc.chunks[2].tokens, [tokens[5]])
+        self.assertSequenceEqual(doc.chunks[0].tokens, tokens[0:3])
+        self.assertSequenceEqual(doc.chunks[1].tokens, tokens[3:5])
+        self.assertSequenceEqual(doc.chunks[2].tokens, [tokens[5]])
 
```

Comment:
`assertListEqual` enforces that both arguments are `list`, while `assertSequenceEqual` does not. Because the first argument is a `Layer`, we use `assertSequenceEqual`.